app/atoms/base/_forms.scss

```diff
@@ -1,22 +0,0 @@
-input {
```

Comment:
Explained on the commit description:

https://github.com/moxie-leean/ng-web-app/pull/55/commits/07ea0d67a7bad58a2c3f543226f7b4e021600a28 <https://github.com/moxie-leean/ng-web-app/pull/55/commits/07ea0d67a7bad58a2c3f543226f7b4e021600a28>

> On Apr 25, 2016, at 5:58 PM, Adam Fenton <notifications@github.com> wrote:
> 
> In app/atoms/base/_forms.scss <https://github.com/moxie-leean/ng-web-app/pull/55#discussion_r61004391>:
> 
> > @@ -1,22 +0,0 @@
> > -input {
> Why remove this file?
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub <https://github.com/moxie-leean/ng-web-app/pull/55/files/07ea0d67a7bad58a2c3f543226f7b4e021600a28#r61004391>